src/main/java/wolox/training/models/User.java

```diff
@@ -0,0 +1,62 @@
+package wolox.training.models;
+
+import java.time.LocalDate;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+import lombok.AccessLevel;
+import lombok.Data;
+import lombok.Setter;
+import wolox.training.exceptions.BookAlreadyOwnedException;
+import wolox.training.exceptions.BookNotOwnedException;
+
+@Entity
+@Data
+@Table(name = "users")
+public class User {
+
+    @Id
+    @Setter(AccessLevel.PRIVATE)
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    @Column(nullable = false)
+    private String username;
+
+    @Column(nullable = false)
+    private String name;
+
+    @Column(nullable = false)
+    private LocalDate birthDate;
+
+    @ManyToMany
```

Comment:
Just a comment about this ManyToMany relationship: When we don't set a cascade type, it will take no operations as cascades. This means that if we re-read an element from the database, its child won't be recover.
In this scenario we need the cascade type REFRESH and MERGE. 

Persist says: If you persist the Users instance to the DB, then persist every item that this collection has in the DB as well.

Delete says: If you delete the Users instance from the DB, then delete every item that this collection has in the DB as well.

Merge says: If you changed the in-memory values of the collection, then merge this changes to the DB so these are reflected there.

Refresh says: If something changed in the database values of the collection, then refresh those changes in the in-memory values.

To read more about the cascade types this link could be useful: https://www.baeldung.com/jpa-cascade-types

Reply:
Awesome! read the source too. thanks for taking the time to summarize it for me.